src/event-cache/EventCache.ts

```diff
@@ -80,6 +80,15 @@ export class EventCache {
         }
     };
 
+    /**
+     * Returns the sampling decision of a session.
+     *
+     * @returns boolean
```

Comment:
nit: Either omit the `@returns` tag, or describe the return value.

By convention we don't typically use the `@returns` tag when the return value is already specified in the title. I believe this convention conforms with [Google's Java code guidelines](https://google.github.io/styleguide/javaguide.html#s7.1.1-javadoc-multi-line) -- which we try to use in this repo -- but violates [Oracle's JavaDoc guidelines](https://www.oracle.com/technical-resources/articles/java/javadoc-tool.html#@return).